stubs/redis/redis/client.pyi

```diff
@@ -477,10 +477,10 @@ class Redis(Generic[_StrType]):
     def mget(self, keys: Union[_Key, Iterable[_Key]], *args: _Key) -> List[Optional[_StrType]]: ...
     def mset(self, mapping: Mapping[_Key, _Value]) -> Literal[True]: ...
     def msetnx(self, mapping: Mapping[_Key, _Value]) -> bool: ...
-    def move(self, name, db): ...
-    def persist(self, name): ...
-    def pexpire(self, name, time): ...
-    def pexpireat(self, name, when): ...
+    def move(self, name: _Key, db: int) -> bool: ...
+    def persist(self, name: _Key) -> bool: ...
+    def pexpire(self, name: _Key, time: Union[int, timedelta]) -> Literal[1, 0]: ...
+    def pexpireat(self, name: _Key, when: Union[int, datetime]) -> Literal[1, 0]: ...
```

Comment:
Just for future reference: PEP 604 can now be used in Python, meaning the union can be written as `int | timedelta` or `int | datetime`.